src/main/java/com/opentable/metrics/JettyServerMetricsConfiguration.java

```diff
@@ -72,9 +72,10 @@
             super(metricRegistry,
                 32, 32, // Default number of threads, overridden in otj-server EmbeddedJetty
                 30000,  // Idle timeout, irrelevant since max == min
-                new BlockingArrayQueue<>(qSize, // Initial queue size
-                    8, // Expand increment (irrelevant; initial == max)
-                    qSize)); // Upper bound on work queue
+                // NB: we originally wished to size this queue, but unfortunately Jetty leaks resources
+                // when you throw RejectedExecutionException due to a full work queue.
```

Comment:
As long as this queue is instrumented (which I assume it is?), I'm ok with this.

Reply:
Indeed it is